### DISCUSSION TITLE
GUI Bugs in 0.4.1 and 0.3.2

### DISCUSSION THREAD

**Discussion by Samuel-Gebe** [linien-org/linien#382] [Q&A] [answered]
Hello, I am facing some issues with the monitor display in version 0.4.1 and 0.3.2. In version 0.4.1, their is no error signal or control voltage displayed on the monitor. In version 0.3.2, the error signal is only displayed when my ramp amplitude is smaller than 0.1. I don't think this would cause a problem but I am outputting 0 V for my modulation amplitude at 0 MHz and performing modulation and demodulation without the red pitaya. Any information would be greatly appreciated.

**Comment by Samuel-Gebe** [comment]
I was using the wrong os version of the red pitaya. I switched to version 1.0 and it seemed to fix the problem

**Reply by bleykauf** [reply, marked as answer]
Yes, only the 1.x releases are compatible with both versions of RedPitaya OS.
